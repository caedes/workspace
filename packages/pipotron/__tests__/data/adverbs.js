import adverbs from '../../src/data/adverbs';

describe('adverbs', () => {
  it('returns the adverbs', () => {
    expect(adverbs).toMatchSnapshot();
  });
});
