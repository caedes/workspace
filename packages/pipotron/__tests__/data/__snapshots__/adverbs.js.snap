// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adverbs returns the adverbs 1`] = `
Array [
  "à long terme",
  "pour longtemps",
  "à l'avenir",
  "pour le futur",
  "depuis longtemps",
  "à court terme",
  "rapidement",
  "dans une perspective correcte",
  "avec toute la prudence requise",
  "de toute urgence",
  "même si ce n'est pas facile",
  "même si nous devons en tirer des conséquences",
  "très attentativement",
  "avec beaucoup de recul",
  "parce que la nature a horreur du vide",
  "parce que nous ne faisons plus le même métier",
  "toutes choses étant égales par ailleurs",
  "et déjà en notre possession",
  "en prenant toutes les précautions qui s'imposent",
  "si l'on veut s'en sortir un jour",
  "parce qu'il est temps d'agir",
  "parce qu'il s'agit de notre dernière chance",
  "parce que les mêmes causes produisent les mêmes effets",
  "parce que nous le valons bien",
]
`;
